### DISCUSSION TITLE
v0.42.3

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#5033] [Announcements]
Patch release with the following fixes/quality of life changes. This being a patch fix, there are no API changes.

## What's Changed

- Fix module vtkImageDilateErode3D not found error by @Arnav-2004  #5020
- Set Google.Spacing NO in .vale.ini by @tkoyama010 #5027
- Update geovista integration test by @bjlittle #4957
- Fix GeoVista's CI error due to image regerssion test by @tkoyama010 #4970

**Full Changelog**: https://github.com/pyvista/pyvista/compare/v0.42.2...v0.42.3

<hr /><em>This discussion was created from the release <a href='https://github.com/pyvista/pyvista/releases/tag/v0.42.3'>v0.42.3</a>.</em>

**Comment by alexunivlr** [comment]
Hi thanks for the release. But When I update my numpy version Igot an AttributeError: module 'numpy' has no attribute 'bool'. Do you know if there is a patch release for the last numpy version?
thanks

**Reply by tkoyama010** [reply]
@alexunivlr Thanks for comment. https://github.com/pyvista/pyvista/issues/4559 may help you.
